client/landing/stepper/declarative-flow/newsletter.ts

```diff
@@ -58,6 +60,9 @@ export const newsletter: Flow = {
 				case 'processingFake':
 					return navigate( 'launchpad' );
 
+				case 'launchpad':
```

Comment:
It looks like we are using `goNext` for our escape to admin functionality. With a term like `goNext` we would first expect this to be what determines the next screen the launchpad flow goes to. However, its also a switch case dependent on the current step - where our LaunchPad wont just flow to 1 specific screen at any point in time, but whichever task the user clicks. 

So I assume out LaunchPad task buttons will be more like links to other stepper pages/flows with specific query params, as opposed to utilizing the `goNext` function directly? The `goNext` on those specific pages would then lead back to launchpad?

With that in mind I think it could make sense to utilize `goNext` for the "go to admin" functionality.  

Am I understanding this correctly?

Reply:
> So I assume out LaunchPad task buttons will be more like links to other stepper pages/flows with specific query params, as opposed to utilizing the goNext function directly to get there?

Correct! That's exactly how I've been envisioning it in my head.

Reply:
You can see examples here https://github.com/Automattic/wp-calypso/blob/21495100815bd6935ae629f3f6480149357cb35e/client/landing/stepper/declarative-flow/site-setup-flow.ts#L476-L492